packages/addon/src/intent_resolver.ts

```diff
@@ -58,22 +66,22 @@ function createIcon(icons: Icon[] | undefined): HTMLElement {
   return img
 }
 
-const fillList = (ai: AppIntent[], intent: string, callback: (s: SingleAppIntent) => void) => {
+const fillList = (ai: AppIntent[], intent: string, callback: (s: IframeResolveActionPayload) => void) => {
```

Comment:
Update to FDC3UserInterface messages and switch to using the top-level type rather than the payload type